osu.Game/Users/UserPanel.cs

```diff
@@ -61,6 +63,9 @@ protected UserPanel(APIUser user)
         [Resolved]
         protected OsuColour Colours { get; private set; } = null!;
 
+        [Resolved]
+        private MultiplayerClient multiplayerClient { get; set; } = null!;
```

Comment:
I was going to say this should probably be a `MultiplayerClient?` but it's defined in `OsuGameBase`... May be okay, though I'd want to know others' opinions.

Reply:
With a class this general I'd agree to keep coupling as loose as can get, so agree with making nullable (even if it's technically always available). This may even be a concern for tournament client, haven't tested.